saleor/warehouse/availability.py

```diff
@@ -62,7 +63,14 @@ def check_stock_quantity_bulk(
         variant_stocks[stock.product_variant_id].append(stock)
 
     insufficient_stocks: List[InsufficientStockData] = []
-    for variant, quantity in zip(variants, quantities):
+    for variant, input_quantity in zip(variants, quantities):
+
+        quantity = input_quantity
```

Comment:
Maybe I am missing something but is there any prons to have two quantity variables here? `quantity` and `input_quantity` ? Looks like this assignment is not needed and we can take from `for` a normal `quantity` (as previously)